src/Stratis.Features.Diagnostic/Stratis.Features.Diagnostic.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Perhaps target this to .net core as we will be moving over to it soon?

Reply:
yes

Reply:
I tried with dotnetcore 2.2 but gave me errors complaining about StratisD not compatible, running 2.1
I'll move this project to netcore2.2 when netcore port will be merged

Reply:
@MithrilMan We need to still change this to Core 2.1